net-irc/hexchat/hexchat-2.12.4.ebuild

```diff
@@ -60,8 +60,10 @@ PATCHES=(
 )
 
 src_prepare() {
+	if [[ ${PV} == "9999" ]]; then
+		eautoreconf
+	fi
 	default
-	eautoreconf
```

Comment:
Please revert this.  The patch listed in PATCHES updates configure.ac, so eautoreconf needs to be called unconditionally.